public/app/features/alerting/unified/components/admin/AlertmanagerConfig.tsx

```diff
@@ -84,80 +99,39 @@ export default function AlertmanagerConfig(): JSX.Element {
         dataSources={alertManagers}
       />
       {loadingError && !loading && (
-        <Alert severity="error" title="Error loading Alertmanager configuration">
-          {loadingError.message || 'Unknown error.'}
-        </Alert>
+        <>
+          <Alert severity="error" title="Error loading Alertmanager configuration">
+            {loadingError.message || 'Unknown error.'}
+          </Alert>
+
+          <AlertmanagerConfigSelector
```

Comment:
I think we should render this selector only in case of grafana alert manager

Reply:
You're right, fixed it.